### PR TITLE
[open-graph-scraper] Add customMetaTags field to Options and Update type test

### PR DESCRIPTION
**Pull request by roseline124**
- Add `customMetaTags` field to `Options` type and also add `CustomMetaTag` interface
- Update Test
  - fix miss spell: `$expectType` -> `$ExpectType` (it's case sensitive 😂)
  - update type of result field in `SuccessResult` type (the type has changed in #50978) 
  - update type of result field in `ErrorResult` type

thank you :)

---

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: 
- I referred these urls
  - [customMetaTags in document](https://github.com/jshemas/openGraphScraper#custom-meta-tag-example)
  - [customMetaTags in source code](https://github.com/jshemas/openGraphScraper/blob/857245d8753380ce9adf0c1956edff06b45b523a/lib/extract.js#L13)

### CODE REVIEW

File: types/open-graph-scraper/index.d.ts [hunk 1]
```diff
@@ -262,6 +268,8 @@ declare namespace run {
         url: string;
         /** Sets the options used by `validator.js` for testing the URL. */
         urlValidatorSettings?: URLValidatorSettings;
+        /** Add custom meta tags if there are no what you want in open graph properties type */
```
Comment (ffflorian) [inline types/open-graph-scraper/index.d.ts:271]: Let's use the official description:


```suggestion
        /** Here you can define custom meta tags you want to scrape. */
```

File: types/open-graph-scraper/open-graph-scraper-tests.ts [hunk 2]
```diff
@@ -7,24 +7,24 @@ const options: openGraphScraper.Options = {
...
+        result.modifiedTime; // $ExpectType string | undefined
+        result.customValue; // $ExpectType string | undefined
     } else {
         const { error, result } = data;
-        error; // $expectType true
-        result; // $expectType ErrorResult
-        result.errorDetails; // $expectType Error
+        error; // $ExpectType true
+        result; // $ExpectType { error: string; errorDetails: Error; success: false; }
+        result.errorDetails; // $ExpectType Error
```
Comment (ffflorian) [inline types/open-graph-scraper/open-graph-scraper-tests.ts:28]: Good catch with all the `ExpectError` 🙂

**Review by ffflorian — CHANGES_REQUESTED** [review]

File: types/open-graph-scraper/index.d.ts (hunk 1 above)
Comment (roseline124) [reply types/open-graph-scraper/index.d.ts:271]: Thanks! :)

**Review by ffflorian — APPROVED** [review]
Thanks :)

**Comment by roseline124** [comment]
Ready to merge

**Pull request merged**